site/content/docs/5.1/components/sticker.md

```diff
@@ -83,8 +83,22 @@ Since stickers only provide a container, accessibility becomes specific to the s
 </div>
 {{< /example >}}
 
-## Sass
+## CSS
 
 ### Variables
 
+{{< added-in "5.2.0" >}}
+
+As part of Boosted's evolving CSS variables approach, stickers now use local CSS variables on `.sticker` for enhanced real-time customization. Values for the CSS variables are set via Sass, so Sass customization is still supported, too.
+
+{{< scss-docs name="sticker-css-vars" file="scss/_sticker.scss" >}}
+
+`sm` and `lg` modifier classes are used to update the value of these CSS variables as needed for **small** and **large** stickers:
+
+{{< scss-docs name="sticker-sm-css-vars" file="scss/_sticker.scss" >}}
+
+{{< scss-docs name="sticker-lg-css-vars" file="scss/_sticker.scss" >}}
+
+### Sass variables
```

Comment:
Just for info @isabellechanclou and @louismaximepiton in the other (or next) PRs regarding CSS custom properties, don't forget to use the new doc structure (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1251/commits/8623bdbb3d1eb09eff5362d4cb004f74b8d83cc9).

Reply:
Ah yup, didn't saw it, Thanks ! 😄